src/main/scala/mesosphere/marathon/core/deployment/impl/DeploymentManagerActor.scala

```diff
@@ -138,8 +138,8 @@ class DeploymentManagerActor(
   val runningDeployments: mutable.Map[String, DeploymentInfo] = mutable.Map.empty
   val deploymentStatus: mutable.Map[String, DeploymentStepInfo] = mutable.Map.empty
 
-  private[this] val runningDeploymentsMetric = Metrics.minMaxCounter(ServiceMetric, getClass, "currentDeploymentCount")
-  private[this] val totalDeploymentsMetric = Metrics.minMaxCounter(ServiceMetric, getClass, "deploymentCount")
+  private[this] val runningDeploymentsMetric = Metrics.atomicGauge("marathon.deployments.running")
```

Comment:
@wavesoft spreadsheet indicates this should be marathon.deployments.active

Reply:
prefer `marathon.deployments.active.count`

Reply:
@kensipe, I thought we agreed on following the naming guidelines that can be found on the Prometheus website. In case of counters, it suggests suffixing with `.total`, and in case of gauges, there is no prefix.

I like `active` part better. Thank you, guys!